packages.json

```diff
@@ -2,6 +2,12 @@
   "packages": {
     "plugins": [
     {
+    	"name": "Xclip",
+    	"url": "https://github.com/sebastianv1/Xclip",
+    	"description": "Multilevel Clipboard that saves your past 10 copies and allows you to paste them inside Xcode.",
+    	"screenshot": "https://github.com/sebastianv1/Xclip/blob/master/XclipScreenshot.png"
```

Comment:
Please use the raw link to the image, we won't be able to load it this way

(https://raw.githubusercontent.com/sebastianv1/Xclip/master/XclipScreenshot.png)